src/esri-loader.ts

```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright 2017 Esri
+  Copyright 2019 Esri
```

Comment:
i'd recommend either leaving this `Copyright 2017` or switching to `Copyright 2017-2019`

more info: https://github.com/Esri/contributing/issues/21